rusty_pizza_server/src/money.rs

```diff
@@ -0,0 +1,383 @@
+use std::fmt::{self, Display, Formatter};
+use std::ops::{Add, Mul, Sub};
+
+#[derive(Debug, PartialEq, Eq, Hash, Copy, Clone)]
+pub struct Money {
+    cents: u32,
+}
+
+impl Money {
+    pub fn new(euros: u32, cents: u8) -> Money {
+        Money {
+            cents: euros * 100 + cents as u32,
```

Comment:
I also thought about this while implementing it. If `cents` would be limited with a check then the return type of the function would be something like `Result<Money, MoneyError>`. This would require the calling code to handle that error. For simplicity reasons I decided to not do this and instead simply allow to pass up to 256 cents allowing for simpler function usage. Do you agree?

Reply:
I agree with you, but I think we should add a comment.

Reply:
Added.